src/_init_openblas.c

```diff
@@ -14,25 +12,9 @@ get_config(PyObject *self, PyObject *args) {
     return PyUnicode_FromString(config);
 }
 
-PyObject*
-open_so(PyObject *self, PyObject *args) {
-    const char *utf8 = PyUnicode_AsUTF8(args);
-    if (utf8 == NULL) {
-        return NULL;
-    }
-    void *handle = dlopen(utf8, RTLD_GLOBAL | RTLD_NOW);
-    if (handle == NULL) {
-        PyErr_SetString(PyExc_ValueError, "Could not open SO");
-        return NULL;
-    }
-    Py_RETURN_TRUE; 
-}
-
```

Comment:
It's not necessary to dlopen this globally for numpy to find the symbols.
Just importing openblas is enough.

Reply:
> Just importing openblas is enough.

How does this work with NumPy? Could you link to your numpy branch?

Reply:
Ahh, got it
```
git clone https://github.com/mayeut/numpy.git numpy
        cd numpy
        git checkout openblas-wheel

```